public/app/features/explore/RichHistory/RichHistory.tsx

```diff
@@ -35,7 +35,9 @@ interface RichHistoryProps extends Themeable {
   activeDatasourceInstance: string;
   firstTab: Tabs;
   exploreId: ExploreId;
+  height: number;
   deleteRichHistory: () => void;
+  onClose: () => void;
 }
```

Comment:
new props:
- height 0 so the slider can adjust its height based on height of drawer. 
- onClose - to close rich history drawer by new **x** button